### ISSUE TITLE
Add video links - youtube for livestreams and fb for meetups

### ISSUE THREAD

**Issue by TomasVotruba** [pehapkari/pehapkari.cz#69]
Here: https://github.com/TroiaStudio/YoutubeAPI

**Comment by TomasVotruba** [comment]
Here: https://github.com/TroiaStudio/YoutubeAPI

**Comment by TomasVotruba** [comment]
I'm finally getting to this, thanks for waiting :)

Any chance you'd be albe to fetch all FB videos from our page?
https://www.facebook.com/pg/pehapkari/videos/?ref=page_internal

**Comment by TomasVotruba** [comment]
@JanGalek Thanks for the package :+1:  I've used some parts of it in #103, mainly setup of API was much easier :) it has weird conter-intuitive REST design, but that's Youtube's fault.  

You'll be able to see it live in 10 mins: https://pehapkari.cz/prehraj-si-prednasku/

Now the harder part, FB videos

**Comment by JanGalek** [comment]
@TomasVotruba no problem :), Have I look at FB videos ? :)

**Comment by TomasVotruba** [comment]
I've checked it. The API is quite ok, I managed to figure out 2 endpoints: https://github.com/pehapkari/pehapkari.cz/blob/e5a70d3e101932b3fbf3166c8236200bbbbfcc81/packages/Youtube/src/Command/ImportVideosCommand.php#L99-L111

The main annoyance is that I need special "page access" token, that required business confirmation = upload founding, contract, IDs, phones bills, videos of how the token will be usedetc. + super shitty UX.

Also, they have 5 days to answer it, and I've found out just yesterday I misses to push 3 buttons :D
